src/client/common/cy/layout/index.js

```diff
@@ -1,3 +1,5 @@
+//Unused For Now
```

Comment:
Please add a comment saying this is produces better results for most layouts that do not have compound support.